Update reduce-css-calc to v2.x.x possible?

When using `recharts` with [Gatsby.js](https://github.com/gatsbyjs/gatsby) v2 `reduce-css-calc` v1.3.0 (required by `recharts`) is overwritten by v2.1.4 breaking the build:

```
ERROR  Failed to compile with 1 errors

This dependency was not found:

* fs in ./node_modules/reduce-css-calc/dist/parser.js

To install it, you can run: npm install --save fs

ERROR in ./node_modules/reduce-css-calc/dist/parser.js
Module not found: Error: Can't resolve 'fs' in 'E:\mui-gatsby-markdown-fr\node_modules\reduce-css-calc\dist'
 @ ./node_modules/reduce-css-calc/dist/parser.js 730:17-30
 @ ./node_modules/reduce-css-calc/dist/index.js
 @ ./node_modules/recharts/es6/component/Text.js
 @ ./node_modules/recharts/es6/index.js
```

--- a/package.json
+++ b/package.json
@@ -64,7 +64,7 @@
     "react-resize-detector": "^4.2.1",
     "react-smooth": "^1.0.5",
     "recharts-scale": "^0.4.2",
-    "reduce-css-calc": "^1.3.0"
+    "reduce-css-calc": "^2.1.7"
   },
   "devDependencies": {
     "@babel/cli": "^7.6.4",

